src/Open3D/Utility/Eigen.h

```diff
@@ -117,5 +119,12 @@ Eigen::Matrix3d RotationMatrixX(double radians);
 Eigen::Matrix3d RotationMatrixY(double radians);
 Eigen::Matrix3d RotationMatrixZ(double radians);
 
+/// Color conversion from double [0,1] to uint8_t 0-255; this does proper
+/// clipping and rounding
+Eigen::Vector3uint8 ColorToUint8(Eigen::Vector3d color);
```

Comment:
From https://eigen.tuxfamily.org/dox/TopicFunctionTakingEigenTypes.html, it looks like the preferred way is to pass Eigen objects with `const &`. Same for the two `ColorToDouble` functions below.

Reply:
good catch